dev-util/itstool/files/itstool-2.0.7-missing-translation-fix.patch

```diff
@@ -0,0 +1,31 @@
+From e9b053be5c50c2bd69442a8484a4c9c371bc3a45 Mon Sep 17 00:00:00 2001
```

Comment:
Always include the link to the upstream commit at the top of patches, but also include links to any relevant bugs or PRs or ...

Reply:
> Always include the link to the upstream commit at the top of patches, but also include links to any relevant bugs or PRs or ...

Sorry about that, I committed the unedited patch by mistake.